### PR TITLE
fix: repository port usage

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #1041 

Fixed cases with repository port:

1) path trimming is necessary when repo port is set (for now, I've only fixed it for files repository and added corresponding tests)
2) repository with port should not be accessible by default port

### CODE REVIEW

File: src/main/java/com/artipie/SliceFromConfig.java [hunk 1]
```diff
@@ -116,16 +116,12 @@ static Slice build(
         );
         switch (cfg.type()) {
             case "file":
-                slice = trimIfNotStandalone(
-                    settings, standalone,
-                    new FilesSlice(cfg.storage(), permissions, auth)
+                slice = new TrimPathSlice(
```
Comment (dgarus) [inline src/main/java/com/artipie/SliceFromConfig.java:119]: We should use the same approach for all slices.

File: src/main/java/com/artipie/http/ArtipieRepositories.java [hunk 2]
```diff
@@ -75,21 +75,27 @@ public Slice slice(final Key name, final boolean standalone) {
...
+        final int port
     ) {
         return new RepositoriesFromStorage(this.http, storage).config(name.string()).thenCombine(
             StorageAliases.find(storage, name),
-            (config, aliases) -> new SliceFromConfig(
-                this.http, this.settings,
-                config, aliases, standalone
-            )
+            (config, aliases) -> {
+                Slice res = new SliceSimple(new RsRepoNotFound(name));
```
Comment (dgarus) [inline src/main/java/com/artipie/http/ArtipieRepositories.java:90]: Think it'll be better to use `else` brunch to create `SliceSimple`.

**Review by dgarus — CHANGES_REQUESTED** [review]

**Comment by olenagerasimova** [comment]
@dgarus all fixed and todos added. Please, take a look

**Review by dgarus — APPROVED** [review]
@olenagerasimova thanks!

**Pull request merged**